build.gradle

```diff
@@ -41,20 +33,23 @@ project.ext {
     junitVintageVersion = '4.12.3'
     junitJupiterVersion = '5.0.3'
     junitPlatformVersion = '1.0.3'
+    intellijAnnotationsVersion = '12.0'
 }
 
 dependencies {
     compile group: 'io.reactivex.rxjava2', name: 'rxjava', version: rxJavaVersion
     compile group: 'io.netty', name: 'netty-handler', version: nettyVersion
     compile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion, classifier: 'linux-x86_64'
-    compile group: 'com.google.dagger', name: 'dagger', version: daggerVersion
     compile group: 'com.google.guava', name: 'guava', version: guavaVersion
     compile group: 'ch.qos.logback', name: 'logback-classic', version: logbackVersion
     compile group: 'org.jctools', name: 'jctools-core', version: jcToolsVersion
 
-    apt group: 'com.google.dagger', name: 'dagger-compiler', version: daggerVersion
+    implementation group: 'com.google.dagger', name: 'dagger', version: daggerVersion
+    annotationProcessor group: 'com.google.dagger', name: 'dagger-compiler', version: daggerVersion
 
     testCompile group: 'org.eclipse.paho', name: 'org.eclipse.paho.client.mqttv3', version: '1.2.0'
+    compileOnly  group: 'com.intellij', name: 'annotations', version: intellijAnnotationsVersion
```

Comment:
We don't use IntelliJ annotations, we use our own @NotNull, @Nullable annotations (maybe we should change this, but in another PR)
This has nothing to do with annotation processing

Reply:
please remove this